commerce_coordinator/apps/frontend_app_payment/exceptions.py

```diff
@@ -6,3 +6,9 @@ class InvalidOrderPayment(APIException):
     status_code = 409
     default_detail = 'Your requested payment does not belong to this payment'
     default_code = 'invalid_order_payment'
+
+
+class UnexpectedPayment(APIException):
+    status_code = 500
+    default_detail = 'Received Unexpected payment data'
+    default_code = 'unexpected_payment_details'
```

Comment:
are these public? (thus localized in some way?)

Reply:
@pshiu, This error is also not in Spec. We are using this error if we get unexpected data from Titans system.